pwa/src/templates/templateParts/dependenciesTemplates/networkDependenciesTemplate/NetworkDependenciesTemplate.module.css

```diff
@@ -0,0 +1,24 @@
+:export {
+  layerColorInteractie: --web-app-layer-color-interaction;
+  layerColorProces: --web-app-layer-color-process;
+  layerColorIntegratie: --web-app-layer-color-integration;
+  layerColorService: --web-app-layer-color-services;
+  layerColorData: --web-app-layer-color-data;
+  borderLayerColorInteractie: --web-app-border-layer-color-interaction;
+  borderLayerColorProces: --web-app-border-layer-color-process;
+  borderLayerColorIntegratie: --web-app-border-layer-color-integration;
+  borderLayerColorService: --web-app-border-layer-color-services;
+  borderLayerColorData: --web-app-border-layer-color-data;
```

Comment:
I suggest to remove the borders altogether to (1) remove complexity in themes and (2) not introduce another color coupled to a layer which is _only_ visible in the relations view.